CommunicationLayer/CommDeviceControl/RadioConnectionService.cpp

```diff
@@ -1,8 +1,18 @@
 #include "RadioConnectionService.h"
+#include <QDebug>
+
+namespace
+{
+   const QString DEFAULT_PORT_NAME = "/dev/ttyAMA0";
```

Comment:
While this works fine for the deadline coming up. I personally think we should start using the QSettings class and have a lot of this type of stuff resource configurable.